mobile/src/store/uam/actions.ts

```diff
@@ -27,12 +28,18 @@ const deleteGroup = createAsyncThunk<
   number,
   GroupsDeleteRequestParamDto,
   AsyncThunkConfig
->(ActionType.DELETE_GROUP, async (payload, { extra }) => {
+>(ActionType.DELETE_GROUP, async (payload, { extra, getState, dispatch }) => {
+  const currentUser = getState().auth.user?.id;
   const { groupsApi } = extra;
-
+  const { id } = payload;
+  const groupsInfo = await groupsApi.getById(payload);
```

Comment:
Don't we store groups info in store?

Reply:
At the UAM store  we have  groups,  but without userIds...userIds for some group we can get only from fetch /groups/:id. Maybe a bad name...better will be: groupInfo